aemedge/blocks/columns/columns.js

```diff
@@ -1,3 +1,91 @@
+import {
+  decorateMain,
+} from '../../scripts/scripts.js';
+
+import {
+  loadBlocks,
+} from '../../scripts/aem.js';
+
+/**
+ * Loads a fragment.
+ * @param {string} path The path to the fragment
+ * @returns {HTMLElement} The root element of the fragment
+ */
+async function loadFragment(path) {
```

Comment:
use `loadFragment` method of` ../fragment/fragment.js` instead of custom method

Reply:
thanks, done